### PR TITLE
API spec update, client demo with Stainless SDK

### PR DESCRIPTION
**Pull request by yanxi0830**
1. Update openapi generator to generate `*RequestWrapper` & OpenAPI spec. 
2. Demo client scripts with Stainless SDK. 

https://github.com/user-attachments/assets/e242344d-f764-4ddb-b109-ad2fcad859f8

https://github.com/user-attachments/assets/0dfb63a4-8575-4e43-946a-f0321a62befa

### CODE REVIEW

File: rfcs/openapi_generator/pyopenapi/generator.py [hunk 1]
```diff
@@ -468,12 +468,14 @@ def _build_operation(self, op: EndpointOperation) -> Operation:
             builder = ContentBuilder(self.schema_builder)
             first = next(iter(op.request_params))
             request_name, request_type = first
+
+            from dataclasses import make_dataclass
+
             if len(op.request_params) == 1 and "Request" in first[1].__name__:
```
Comment (ashwinb) [inline rfcs/openapi_generator/pyopenapi/generator.py:474]: can you just remove the `if` completely? Then you will get a class named `FooRequestRequest` but that's OK. It is an implementation detail anyway.

File: llama_toolchain/agentic_system/client_sdk.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import fire
```
Comment (ashwinb) [inline llama_toolchain/agentic_system/client_sdk.py:1]: @hardikjshah do you have any opinions on how we should situate these files?

File: rfcs/openapi_generator/pyopenapi/generator.py (hunk 1 above)
Comment (yanxi0830) [reply rfcs/openapi_generator/pyopenapi/generator.py:474]: I tried remove the `if`, but it needs some work to generate the `*Wrapper` class for the `request` class. E.g. https://github.com/meta-llama/llama-stack/commit/838ab91ebf7ea13317f94b0a07e84a271e6b9e20

File: llama_toolchain/agentic_system/client_sdk.py (hunk 2 above)
Comment (yanxi0830) [reply llama_toolchain/agentic_system/client_sdk.py:1]: Another option is to put them alone with the SDK repos: https://github.com/stainless-sdks/llama-stack-python/tree/yanxi0830/dev

Comment (hardikjshah) [reply llama_toolchain/agentic_system/client_sdk.py:1]: yeah , the SDK repo's examples folder is probably best.

Comment (yanxi0830) [reply llama_toolchain/agentic_system/client_sdk.py]: Done (removed these scripts from the PR)! In the meantime (while SDK repo is getting ready), adding the `sdk_examples` to `llama-stack-apps` in https://github.com/meta-llama/llama-stack-apps/pull/66.

**Review by ashwinb — APPROVED** [review]
lg

**Pull request merged**